### ISSUE TITLE
bug: Fix tests/src/kakarot/test_kakarot.py not running since keccak from cairo1 merge

### ISSUE THREAD

**Issue by ClementWalter** [kkrt-labs/kakarot#1071]
See error

```
E           /Users/clementwalter/Documents/sayajin-labs/kakarot/src/kakarot/interpreter.cairo:198:9: (pc=0:21679)
E                   call Sha3.exec_sha3;  // 0x20
E                   ^*****************^
E           /Users/clementwalter/Documents/sayajin-labs/kakarot/src/kakarot/instructions/sha3.cairo:65:24: (pc=0:18306)
E                   let (result) = ICairo1Helpers.library_call_keccak(
E                                  ^*********************************^
E           autogen/starknet/contract_interface/ICairo1Helpers/keccak/1cc3b6a9adce5eadc4f3ce988125697adcafe7b5ffd894f0c011db065f2211ae.cairo:2:31: (pc=0:4508)
E           let (retdata_size, retdata) = library_call(
E                                         ^***********^
E           
E           Traceback (most recent call last):
E             File "/Users/clementwalter/.pyenv/versions/3.10.13/envs/kakarot/lib/python3.10/site-packages/starkware/starknet/common/syscalls.cairo", line 96, in <module>
E               %{ syscall_handler.library_call(segments=segments, syscall_ptr=ids.syscall_ptr) %}
E             File "/Users/clementwalter/Documents/sayajin-labs/kakarot/tests/utils/syscall_handler.py", line 373, in library_call
E               raise ValueError(
E           ValueError: Function selector 0x335a135a69c769066bbb4d17b2fa3ec922c028d4e4bf9d0402e6f7c12b31813 not found in patches.
```

simply do to the fact that the sha3 is now delegated to the cairo1 helper.

## What

Add a patch to compute sha3/keccak using the python function when caller the given function selector.

**Comment by tit0uanf** [comment]
assign

**Comment by enitrat** [comment]
@ClementWalter fixed by #1107 ?

**Comment by ClementWalter** [comment]
just tested and it's solved actually @enitrat @tit0uanf
